src/main/java/io/confluent/connect/jdbc/TimestampIncrementingTableQuerier.java

```diff
@@ -60,15 +60,17 @@
   private Long timestampOffset;
   private String incrementingColumn;
   private Long incrementingOffset = null;
+  private Long timestampDelay;
```

Comment:
This never takes on a `null` value and isn't used as an `Object`, so should really be `long`.